src/deepsparse/yolov8/validation.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+
+from deepsparse import Pipeline
+from deepsparse.yolo.utils import COCO_CLASSES
+from deepsparse.yolov8.utils import DeepSparseDetectionValidator
+from ultralytics.yolo.cfg import get_cfg
+from ultralytics.yolo.utils import DEFAULT_CFG
+
+
+DEEPSPARSE_ENGINE = "deepsparse"
+ORT_ENGINE = "onnxruntime"
+SUPPORTED_DATASET_CONFIGS = ["coco128.yaml", "coco.yaml"]
+
+
+@click.command()
```

Comment:
set context settings to avoid specifying underscore + hyphenated variants of args